src/settings/NumberGeneratorSettings/constants.js

```diff
@@ -0,0 +1,14 @@
+export const NUMBER_GENERATOR_OPTIONS = {
+  USE_BOTH: 'useBoth',
+  USE_GENERATOR: 'useGenerator',
+  USE_TEXT_FIELD: 'useTextField',
+};
+
+export const NUMBER_GENERATOR_SETTINGS_KEY = 'number-generator-settings';
+export const NUMBER_GENERATOR_SETTINGS_SCOPE = 'ui-inventory.number-generator-settings';
```

Comment:
This must be the same as the `scope` part of the permission - `ui-inventory.number-generator-settings.manage`.

Reply:
Added `.manage`

Reply:
It only mattered to the `NUMBER_GENERATOR_SETTINGS_SCOPE`.

Reply:
OK, sorry, reverted key to:
`export const NUMBER_GENERATOR_SETTINGS_KEY = 'number-generator-settings';`